libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts

```diff
@@ -4,9 +4,13 @@ import { CommonModule } from '@angular/common';
 import { ConsultancyRoutingModule } from './consultancy.routing';
 
 import { ConsultancyPageComponent } from './main/consultancy-page/consultancy-page.component';
+import { ElewaConsultancyAboutOneComponent } from './components/elewa-consultancy-about-one/elewa-consultancy-about-one.component';
+import { BannersModule } from '@elewa-website/elements/banners';
+import { MainPageModule } from '@elewa-website/elements/layout/pages/main-page';
 
```

Comment:
Kindly, observe the order of imports; elements imports should appear before local imports